search-books/src/main/java/nix/education/util/SymbolsSolrQuery.java

```diff
@@ -0,0 +1,7 @@
+package nix.education.util;
+
+public class SymbolsSolrQuery {
+    public static final String QUERY = "q";
+    public static final String OR = " OR ";
+    public static final String TRUE = "true";
```

Comment:
Some of the above constants are presented here org.apache.solr.common.params.CommonParams

Reply:
Moved all constants to interface SearchConstants

Reply:
I collect whole constants in SearchConstants interface